README.md

```diff
@@ -65,8 +65,8 @@ module.exports = customService(async function helloWorldService(service) {
   })
 }) 
 ```
-- The library exports a function, `customService`, that expects an async function to initialize and configure the `service`. Optionally can take a schema of the required environment variables (you can find the reference [here][fastify-env]).
-- `service` is a [Fastify instance](https://www.fastify.io/docs/latest/Server/), upon which, you can call `addRawCustomPlugin` method that allows you to add your route. You can use  *service* to register any Fastify plugin, see [here][fastify-ecosystem] for a list of currently available plugins.
+- The library exports a function that returns another functions. Optionally can take a schema of the required environment variables (you can find the reference [here][fastify-env]). The function returned, `customService`, expects an async function to initialize and configure the `service`.
```

Comment:
Maybe we should clarify better this sentence... Is one of the first interesting things a user reads about the plugin and I think it's a bit too technical without giving useful information, for instance: `The library exports a function that returns another functions.` it's true but still a user reading this line won't learn much on how to use the library..

Maybe there's a better way to express this concept?

Reply:
@fredmaggiowski I have recovered the definition from Docs, give me feedback:

"The library exports a function which creates the infrastructure ready to accept the definition of routes and decorators. Optionally can take a schema of the required environment variables you can find the reference [here]. The function returned, `customService`, expects an async function to initialize and configure the `service`. "

Reply:
Sounds good! Let's use this definition!!